synapse/handlers/auth.py

```diff
@@ -1020,15 +1039,17 @@ def generate_access_token(self, user_id, extra_caveats=None):
             macaroon.add_first_party_caveat(caveat)
         return macaroon.serialize()
 
-    def generate_short_term_login_token(self, user_id, duration_in_ms=(2 * 60 * 1000)):
+    def generate_short_term_login_token(
+        self, user_id: str, duration_in_ms: int = (2 * 60 * 1000)
+    ) -> str:
         """
 
         Args:
-            user_id (unicode):
-            duration_in_ms (int):
+            user_id
+            duration_in_ms
 
         Returns:
-            unicode
+            str
```

Comment:
Is this docstring really necessary now that we have the types in the function's definition?

Reply:
Without having additional information it is probably not useful! I'll remove it.